hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/HermesMetricsConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package pl.allegro.tech.hermes.management.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import pl.allegro.tech.hermes.common.di.factories.MetricRegistryFactory;
+import pl.allegro.tech.hermes.common.metric.HermesMetrics;
+import pl.allegro.tech.hermes.common.util.InetAddressHostnameResolver;
+import pl.allegro.tech.hermes.metrics.PathsCompiler;
+
+@Import({
+        pl.allegro.tech.hermes.common.di.factories.MetricRegistryFactory.class,
+        pl.allegro.tech.hermes.common.config.ConfigFactory.class,
+        pl.allegro.tech.hermes.common.metric.counter.zookeeper.ZookeeperCounterStorage.class,
+        pl.allegro.tech.hermes.common.util.InetAddressHostnameResolver.class
+})
+@Configuration
+public class HermesMetricsConfiguration {
```

Comment:
IMO this can be removed as we don't have to use `HermesMetrics` class (it's dedicated for frontend/consumers).
Management is a spring-boot app, so there's micrometer's `MeterRegistry` available in the app context, and so it should be injected...

Reply:
Done